README.md

```diff
@@ -557,9 +557,11 @@ If you'd like `just` to default to listing the recipes in the `justfile`, you ca
 
 ```make
 default:
-  @just --list
+  @just -f {{justfile()}} --list
 ```
 
+Note that the `-f {{justfile()}}` above is needed to make sure the listed are from the current `justfile`. If you executed `just -f /path/to/justfile`, you could list recipes from somewhere else or get a `No justfile found` error.
```

Comment:
I think this could use some work. It's not clear *why* you wouldn't get the right justfile, so I think it should mention something about why you could get the wrong justfile.

Reply:
I think the thing it should mention is that if you're using a justfile with a non-standard name, you might not find it.

Reply:
it's either a non-standard name or a distant path, which was my case. Let me see if I can rephrase it better.